doc/getting-started/gsg_quick_setup.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+# Copyright (C) 2018 Intel Corporation.
+# SPDX-License-Identifier: BSD-3-Clause
+# This script is aim to setup or upgrade sos & uos automatically.
+
+# switch for download uos function
+skip_download_uos=0
+# turn on if you need to setup uos
+upgrade_uos=0
+
+function print_help()
+{
+echo "Usage:"
+echo "Launch this script as: ./flash.sh -t 28100"
+echo "-t to specify clear linux version for upgrading"
+echo "-p to specify a proxy server"
+echo "-m to use swupd mirror url"
+echo "-u to upgrade uos, if you just upgrade sos using this script, you'd better to manually reboot sos then continue to use this function."
```

Comment:
I think I'll wait to do an English edit until after @gvancuts technical issues are addressed. For example this would be better as:
```
echo "-u to upgrade UOS.  After upgrading you must manually reboot the system."
```

Reply:
Rewrite upgrade uos function. Now it can be upgraded directly without checking sos clear linux version.  Also users don't need to reboot device for taking effects after it.